justin-tan/week5/tues/rails-crud-hw/app/views/restaurants/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<h2>Restaurant Detail</h2>
+
+<img src="<%= @restaurant.restaurant_img %>" alt="<%= @restaurant.name %>">
+
```

Comment:
use the image tag here:

`<%= image_tag @restaurant.restaurant_img, alt: @restaurant.name %>`